clusterloader2/testing/load/config.yaml

```diff
@@ -0,0 +1,147 @@
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+- name: Uniform5qps
+  qpsLoad:
+    qps: 5
+steps:
+- measurements:
+  - Identifier: GlobalTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: CreateSaturationPodsTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+# Create RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform100qps
+    objectBundle:
+    - basename: saturation-service
```

Comment:
I don't think you need to call this `saturation-service`. 'service' is enough I think.

Reply:
Same for basename of RC.

Reply:
Done: `saturation-service` -> `service`.